docs/documentation.md

```diff
@@ -558,14 +558,28 @@ no binary available continue working in a temporary directory to avoid failure.
 
 You can control some compilation variables by using environment variables.
 
-#### Disable livirt support
+#### Disable libvirt support
 
 If you wish to compile mgmt without libvirt, you can use the following command:
 
 ```
 GOTAGS=novirt make build
 ```
 
+#### Disable augeas support
+
+If you wish to compile mgmt without augeas support, you can use the following command:
+
+```
+GOTAGS=noaugeas make build
+```
+
+### Combining compile-time flags
```

Comment:
Patch looks perfect, except I think this section needs a fourth ```#```. If not, then the general comment expressed here should be in the `###` "Compilation options" section.